conf/scylla.yaml

```diff
@@ -591,6 +591,14 @@ consistent_cluster_management: true
 # to make sure that trying to create an invalid view causes an error.
 strict_is_not_null_in_views: true
 
+# The Unix Domain Socket the node uses for maintenance socket.
+# The possible options are:
+# * ignore: the node will not open the maintenance socket,
+# * workdir: the node will open the maintenance socket on the path <scylla's workdir>/cql.m,
+#            where <scylla's workdir> is a path defined by the workdir configuration option,
+# * <socket path>: the node will open the maintenance socket on the path <socket path>.
+maintenance_socket: ignore
+
```

Comment:
Why not `workdir`?

Reply:
And no need to include it in scylla.yaml, can just inherit the default from configuration.

Reply:
from .cc

Reply:
It breaks dtests unfortunately, because all clusters in a dtest run are using the same workdir (ccm when creating Scylla configuration, instead of setting `workdir`, they are setting data, commitlog etc directories separately.

I propose we fix this in a follow-up -- we'll need a change in dtest/ccm so each cluster creates its own workdir, and then we can switch this to workdir.